src/dmd/parse.d

```diff
@@ -193,19 +193,13 @@ __gshared PREC[TOKMAX] precedence =
 
 enum ParseStatementFlags : int
 {
-    PSsemi          = 1,        // empty ';' statements are allowed, but deprecated
-    PSscope         = 2,        // start a new scope
-    PScurly         = 4,        // { } statement is required
-    PScurlyscope    = 8,        // { } starts a new scope
-    PSsemi_ok       = 0x10,     // empty ';' are really ok
+    semi          = 1,        // empty ';' statements are allowed, but deprecated
+    scope_        = 2,        // start a new scope
+    curly         = 4,        // { } statement is required
+    curlyscope    = 8,        // { } starts a new scope
```

Comment:
We should consider if we want to enforce the style guide here and make it `curlyScope`.

Reply:
It would also be `semiOk` ...

Reply:
Yes.

Reply:
Ok changed.

Reply:
Not that I'm complaining, but we haven't done that in any other of these PRs. All enum names are still all uppercase.